docs/js/app-config.js

```diff
@@ -27,12 +27,23 @@ define(['angular'], function(angular) {
             'notificationFullURL' : 'notifications'
         })
         .constant('MISC_URLS',{
+            'back2ClassicURL' : null,
             'feedbackURL' : 'https://my.wisc.edu/portal/p/feedback',
             'helpdeskURL' : 'https://kb.wisc.edu/helpdesk/',
             'myuwHome' : 'https://my.wisc.edu',
             'rootURL' : '/uw-frame/',
             'logoutURL' : 'https://github.com/UW-Madison-DoIT/uw-frame',
         })
+        .constant('FOOTER_URLS', [
+          { "url" : "/web/static/myuw-help",
+            "target" : "_blank",
```

Comment:
I would think so. I would be frustrated if I went to read help on a page i was working on and the help took over that page.